modules/uninstall-cluster-logging-operator.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-uninstall.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="uninstall-cluster-logging-operator_{context}"]
+= Uninstalling the {logging}
+
+You can stop log aggregation by deleting the {clo} and the `ClusterLogging` custom resource (CR).
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have access to the *Administrator* perspective of the {product-title} web console.
+
+.Procedure
+
+. Navigate to the *Administration* -> *Custom Resource Definitions* page. Click *ClusterLogging*.
+
+. On the *Custom Resource Definition Details* page, click *Instances*.
+
+. Click the options menu {kebab} next to the instance, then click *Delete ClusterLogging*.
+
+. Navigate to the *Administration* -> *Custom Resource Definitions* page.
+
+. Click the options menu {kebab} next to *ClusterLogging* and select *Delete Custom Resource Definition*.
++
+[NOTE]
```

Comment:
If data loss can occur, I'd recommend changing this to a Warning admonition instead of a Note: https://redhat-documentation.github.io/supplementary-style-guide/#admonitions